src/main/java/de/csbdresden/stardist/StarDist2D.java

```diff
@@ -400,4 +411,59 @@ public static void main(final String... args) throws Exception {
         ij.command().run(StarDist2D.class, true, params);
     }
 
+    private void refresh() {
+        final HashMap<String, Object> params = new HashMap<>();
+
+        params.put("input", input);
```

Comment:
For timelapse input data: should the preview only be for the current frame (how to even get that?) or the entire sequence (potentially slow)?

Reply:
It depends on the actual processing logic of StarDist. Does it probably take into account of whole timeseries of the input image? Then, it will be slow as you expected. How do you handle ROIs over time in this case?